commands/run.sh

```diff
@@ -40,8 +40,16 @@ prebuilt_service_overrides=()
 prebuilt_services=()
 
 # We look for a prebuilt images for all the pull services and the run_service.
+prebuilt_image_override="$(plugin_read_config RUN_IMAGE)"
 for service_name in "${prebuilt_candidates[@]}" ; do
-  if prebuilt_image=$(get_prebuilt_image "$service_name") ; then
+  prebuilt_image=$(get_prebuilt_image "$service_name") || :
```

Comment:
indeed, though I prefer the usage of `true` instead of the more obscure `:`  to be more explicit:

```suggestion
  prebuilt_image=$(get_prebuilt_image "$service_name") || true
```